services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -155,10 +155,25 @@
           "intro": {
             "section": "Section A: Program Information",
             "subsection": "Add BSS entities (A.8)",
+            "spreadsheet": "A_Program_Info",
             "info": [
               {
-                "type": "text",
-                "content": "Enter the names of Beneficiary Support System (BSS) entities that support enrollees in the program for which the state is reporting data."
+                "type": "html",
+                "as": "span",
+                "content": "Enter the names of Beneficiary Support System (BSS) entities that support enrollees in the program for which the state is reporting data. Learn more about BSS entities at "
+              },
```

Comment:
Info: Because `info:` is an array, all of this will be parsed by the [custom HTML parser](https://github.com/CMSgov/mdct-mcr/blob/3671491dfa78d091e1e32883c2a5fdbd49d9aa69/services/ui-src/src/utils/other/parsing.ts#L10-L46) we built.

There are two primary ways to interact with that parser:
1. You can specify `type: "html"` to tell it to check `content:` and parse whatever HTML is in there. But you cannot specify `as:` if you do this because the `as` will be disregarded in favor of whatever HTML you've written in the content. 
2. You can specify something else as `type:`, including any valid html tag or one of our custom types (heading, text, externalLink, internalLink) and they will be handled accordingly.

Because you're specifying both `type: "html"` and `as:` here, the `as` is being disregarded and instead it's just parsing the string you pass in `content:`, which means it renders as text loose inside a div, which can cause accessibility problems.

<img width="413" alt="Screen Shot 2022-10-06 at 17 16 40" src="https://user-images.githubusercontent.com/27705928/194428662-a574d6c2-058c-4ec1-a68b-449eeaa9d17d.png">

`as` can only really effectively be used with `heading` to specify what kind of heading (h1, h2, etc.) you want.

In this case, you want to do either: 

```suggestion
                "type": "html",
                "content": "<span>Enter the names of Beneficiary Support System (BSS) entities that support enrollees in the program for which the state is reporting data. Learn more about BSS entities at </span>"
              },
```

OR

```suggestion
                "type": "span",
                "content": "Enter the names of Beneficiary Support System (BSS) entities that support enrollees in the program for which the state is reporting data. Learn more about BSS entities at "
              },
```

Reply:
I tried both of those options but the text turned gray in both scenarios. I just got rid of the "as" key, is that also viable?